static/css/locus68.css

```diff
@@ -68,8 +68,11 @@ h1, h2 {
 /* stolen from https://www.abeautifulsite.net/how-to-make-rounded-images-with-css */
 .img-circle {
   border-radius: 50%;
+  opacity: 0.5;
```

Comment:
We probably don't want to do this on `.img-circle` since it's a generic class. It'll have consequences like having to change this line https://github.com/TheOrangeOne/locus68/blob/131990f6861701a76dbb8c72edd2f6749bc65be4/static/js/map.js#L306
Which we don't really want to do because else we'll have to change **everywhere** we use `.img-circle`. It's good to keep `.img-circle` as generic as possible so that we can reuse it.

Instead, an approach we could take is to add the opacity to `.select-avatar`: https://github.com/TheOrangeOne/locus68/blob/131990f6861701a76dbb8c72edd2f6749bc65be4/static/css/locus68.css#L226-L229
and the similarly for `.img-circle:hover`, it should be on `.select-avatar:hover`.